FirebaseAppCheck/Tests/Integration/FIRDeviceCheckAPIServiceE2ETests.m

```diff
@@ -14,6 +14,17 @@
  * limitations under the License.
  */
 
+#import <TargetConditionals.h>
+
+// Tests that use the Keychain require a host app and Swift Package Manager
+// does not support adding a host app to test targets.
+#if !SWIFT_PACKAGE
```

Comment:
Why does SWIFT_PACKAGE need to be excluded?  What's changed about this for iOS and tvOS?

Reply:
So this file has one test and it has been disabled for some time. I just saw it was included in the Package.manifest under the exclusions due to host app requirement.

After noticing the tests were disabled, I added a TODO for myself a few months ago.

Reply:
It's not urgent because App Check has excellent test coverage but it would be nice to get this file fixed so the test runs.